osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/utils/ScalerUnitInfo.java

```diff
@@ -16,11 +16,11 @@
     private final int classId;
     private final DataObject scalerUnit;
 
-    public ScalerUnitInfo(final String logicalName, final int classId, final DataObject scalarUnit) {
+    public ScalerUnitInfo(final String logicalName, final int classId, final DataObject scalerUnit) {
```

Comment:
It seems to me this is a wrong modification: scalar sounds perfectly fine. Rename the class to ScalarUnitInfo?

Reply:
yes, I had exactly the same idea, so I changed everything to SCALAR, only to find out that in the openmuc lib, the word SCALER is used... so I changed it all back again....

Reply:
RegisterAttribute.SCALER_UNIT.attributeId());